src/Microsoft.DotNet.Interactive.Tests/LanguageKernelPackageTests.cs

```diff
@@ -496,10 +487,8 @@ public async Task Pound_i_nuget_with_multi_submissions_combines_the_Text_Updates
 
             using var events = result.KernelEvents.ToSubscribedList();
 
-            // For the DisplayedValueUpdated events strip out the restore sources
-            // Verify that they match the expected values
-            events.OfType<DisplayedValueUpdated>()
-                  .Last().FormattedValues
+            events.OfType<DisplayedValueProduced>()
+                  .First().FormattedValues
```

Comment:
Same as above.

Reply:
I think there are two displayValueProduced.
1 for restore sources and 1 for packages loaded.

Reply:
The test isn't loading any packages.

Reply:
Ahh but DoNugetRestore produces two messages always.
1 :  https://github.com/dotnet/interactive/blob/0dcf94198da95e8387295c39ab55a8ea941d9854/src/Microsoft.DotNet.Interactive/KernelSupportsNugetExtensions.cs#L216
2: https://github.com/dotnet/interactive/blob/0dcf94198da95e8387295c39ab55a8ea941d9854/src/Microsoft.DotNet.Interactive/KernelSupportsNugetExtensions.cs#L221